flight_data_collect/drone_communication/mavlink_utils.py

```diff
@@ -34,15 +34,18 @@ def get_mavlink_messages(connect_address):
             msg = _get_mavlink_message(mavlink, msg_type, connect_address)
             if msg and 'ERROR' not in msg:
                 timeout_count = max(0, timeout_count - 1)  # decrement by 1 if timeout_count > 0
-                if msg.get("mavpackettype", "") == mavlink_constants.GPS_RAW_INT and _is_gps_fix(msg):
```

Comment:
are edits to this code block necessary? this seems to be location message-specific logic